addon/components/inputs/select.js

```diff
@@ -86,7 +86,7 @@ export default AbstractInput.extend({
     const enumDef = bunsenModel.items ? bunsenModel.items.enum : bunsenModel.enum
     let renderOptions = get(cellConfig, 'renderer') || {}
     const spreadOptions = get(cellConfig, 'renderer.options')
-    if (spreadOptions) renderOptions = merge(renderOptions, spreadOptions)
+    if (spreadOptions) renderOptions = merge({}, renderOptions, spreadOptions)
```

Comment:
Oh darnit.  Can't believe none of the test caught that.  Will address this Monday.